src/actions/getMyRecipes.ts

```diff
@@ -0,0 +1,28 @@
+import { prisma } from "../lib/prisma";
+import { getAuthenticatedUser } from "./getAuthenticatedUser";
+
+const getMyRecipes = async () => {
+  const user = await getAuthenticatedUser();
+
+  if (!user) {
+    return null;
+  }
+
+  const myRecipe = await prisma.recipe.findMany({
+    where: {
+      userId: user.id,
+      deletedAt: null,
+    },
+    include: {
+      RecipeImage: true,
+      likes: true,
```

Comment:
```suggestion
      _count: {
        select: {
          likes: true,
        },
      },      
```
ここではいいねの数だけほしいので集約関数を用いて取得するようにお願いします！